pkg/audit/verifier_test.go

```diff
@@ -137,22 +47,26 @@ func TestFailingAudit(t *testing.T) {
 	badPieceNums := []int{0, 2, 3, 4}
 
 	ctx := context.Background()
-	auditPkgShares := make(map[int]audit.Share, len(modifiedShares))
+	auditPkgShares := make(map[int]Share, len(modifiedShares))
 	for i := range modifiedShares {
-		auditPkgShares[modifiedShares[i].Number] = audit.Share{
-			PieceNumber: modifiedShares[i].Number,
-			Data:        append([]byte(nil), modifiedShares[i].Data...),
+		auditPkgShares[modifiedShares[i].Number] = Share{
+			PieceNum: modifiedShares[i].Number,
+			Data:     append([]byte(nil), modifiedShares[i].Data...),
 		}
 	}
-	pieceNums, err := auditShares(ctx, 8, 14, auditPkgShares)
+
+	pieceNums, correctedShares, err := auditShares(ctx, 8, 14, auditPkgShares)
 	if err != nil {
 		panic(err)
 	}
+
 	for i, num := range pieceNums {
 		if num != badPieceNums[i] {
 			t.Fatal("expected nums in pieceNums to be same as in badPieceNums")
 		}
 	}
+
+	assert.Equal(t, shares, correctedShares)
```

Comment:
maybe `require` instead?

Reply:
oh, yes, I had a loop before which was using `require` when I've realized that it could be compared directly, no idea why I used `assert`, although the output is the same for this case.

Reply:
I pushed a new commit with this change and realized that there was another `assert` in the test which already exists, hence I've replaced it too.